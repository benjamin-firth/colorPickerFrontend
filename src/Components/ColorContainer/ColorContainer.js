import React from 'react';
import './ColorContainer.scss';
import ColorCard from '../ColorCard/ColorCard';
import PropTypes from 'prop-types';

function ColorContainer() {
  return (
    <section class='color-container'>
      <ColorCard />
      <ColorCard />
      <ColorCard />
      <ColorCard />
      <ColorCard />
    </section>
  );
}

export default ColorContainer;
