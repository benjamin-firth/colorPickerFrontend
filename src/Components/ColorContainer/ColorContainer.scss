.color-card-container {
  display: flex;
  justify-content: center;
}

.generate-palette-button {
  box-shadow: 3px 3px;
  background-color: #CDCDCD;
}
