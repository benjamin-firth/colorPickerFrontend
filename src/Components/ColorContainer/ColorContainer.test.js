import React from 'react';
import { render } from '@testing-library/react';
import ColorContainer from './ColorContainer';

describe('', () => {
  it('should', () => {

  })
});
