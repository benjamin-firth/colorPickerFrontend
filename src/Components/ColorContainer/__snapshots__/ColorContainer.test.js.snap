// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorContainer should match the snapshot 1`] = `
<section
  className="color-container"
>
  <div
    className="color-card-container"
  >
    <ColorCard
      cardColor={
        Object {
          "color": "",
          "id": 1,
          "locked": false,
        }
      }
      lockColor={[Function]}
    />
    <ColorCard
      cardColor={
        Object {
          "color": "",
          "id": 2,
          "locked": false,
        }
      }
      lockColor={[Function]}
    />
    <ColorCard
      cardColor={
        Object {
          "color": "",
          "id": 3,
          "locked": false,
        }
      }
      lockColor={[Function]}
    />
    <ColorCard
      cardColor={
        Object {
          "color": "",
          "id": 4,
          "locked": false,
        }
      }
      lockColor={[Function]}
    />
    <ColorCard
      cardColor={
        Object {
          "color": "",
          "id": 5,
          "locked": false,
        }
      }
      lockColor={[Function]}
    />
  </div>
  <button
    className="generate-palette-button"
    onClick={[Function]}
    type="button"
  >
    Generate New Palette
  </button>
  <PaletteForm
    currentPalette={
      Object {
        "color1": Object {
          "color": "",
          "id": 1,
          "locked": false,
        },
        "color2": Object {
          "color": "",
          "id": 2,
          "locked": false,
        },
        "color3": Object {
          "color": "",
          "id": 3,
          "locked": false,
        },
        "color4": Object {
          "color": "",
          "id": 4,
          "locked": false,
        },
        "color5": Object {
          "color": "",
          "id": 5,
          "locked": false,
        },
      }
    }
  />
</section>
`;
